modules/datadog/variables.tf

```diff
@@ -51,3 +57,9 @@ variable "datadog_secret" {
   description = "Name of the datadog secret in Secrets Manager"
   type        = string
 }
+
+variable "datadog_agent_version" {
```

Comment:
Since its only related to fargate from the description should it refer that in the name or is this version used everywhere?

Reply:
👍